1.4 Test failure TestRSACertificate.test_cert_serial_warning

While testing an upgrade of the GNU Guix python-cryptography package from 1.3.2 to 1.4, I encountered the following failure of ```TestRSACertificate.test_cert_serial_warning[backend0]```.

```
=================================== FAILURES ===================================
____________ TestRSACertificate.test_cert_serial_warning[backend0] _____________

self = <tests.test_x509.TestRSACertificate object at 0x7fffe0fa7978>
backend = <cryptography.hazmat.backends.openssl.backend.Backend object at 0x7ffff3020278>

    def test_cert_serial_warning(self, backend):
        cert = _load_cert(
            os.path.join("x509", "PKITS_data", "certs", "GoodCACert.crt"),
            x509.load_der_x509_certificate,
            backend
        )
    
        with warnings.catch_warnings():
            warnings.simplefilter("always", utils.DeprecatedIn10)
>           with pytest.deprecated_call():
E           TypeError: deprecated_call() missing 1 required positional argument: 'func'

tests/test_x509.py:516: TypeError
============ 1 failed, 83123 passed, 3222 skipped in 164.91 seconds ============
```

The dependency graph is using Python 3.4.3 and Hypothesis 3.1.0. Please let me know what other information I can provide to help debug this failure.

--- a/setup.py
+++ b/setup.py
@@ -58,7 +58,7 @@ else:
     setup_requirements.append("cffi>=1.4.1")
 
 test_requirements = [
-    "pytest",
+    "pytest>=2.9.0",
     "pretend",
     "iso8601",
     "pyasn1_modules",

